packages/documentation-framework/components/tableOfContents/tableOfContents.css

```diff
@@ -2,8 +2,8 @@
   align-self: flex-start;
   position: sticky;
   width: calc(100% + var(--pf-v6-c-page__main-section--PaddingLeft) + var(--pf-v6-c-page__main-section--PaddingRight));
-  background-color: var(--pf-t--global--background--color--primary--default);
-  z-index: 501;
+  background-color: var(--pf-t--global--background--color--200);
+  z-index: var(--pf-t--global--z-index--sm);
```

Comment:
I would probably try "201" here. We use `--sm` (200) for random little component things that need to stack on top of other things. We use `--md` (300) for sticky sections. `--lg` (400) for backdrop, `--xl` (500) for modal. My guess is 201 might interfere the least with the TOC overlapping examples?